apps/beeai-cli/src/beeai_cli/commands/resource_server.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2025 © BeeAI a Series of LF Projects, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+from beeai_cli.async_typer import AsyncTyper, console
+from beeai_cli.configuration import Configuration
+
+app = AsyncTyper()
+
+config = Configuration()
+
+
+@app.command("show")
+def show_server():
+    active = config.auth_manager.get_active_resource()
+    if not active:
+        console.print("No active server!!!\n")
+        return
+    console.print(f"Active server: {active}\n")
+
+
+@app.command("list")
+def list_server():
+    resources = config.auth_manager.config.get("resources", {})
+    if not resources:
+        console.print("No servers logged in. Run `beeai login` first.\n")
+        return
+    console.print("Known servers:")
+    for res in resources:
+        marker = " (active)" if res == config.auth_manager.get_active_resource() else ""
+        console.print(f"{res}{marker}\n")
+
+
+@app.command("switch")
+def switch_server():
+    resources = config.auth_manager.config.get("resources", {})
+    if not resources:
+        console.print("No server logged in. Run `beeai login` first.\n")
```

Comment:
Use consistent formatting with other console messages, e.g. `[green]...[/green]` for commands.